Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -62,18 +62,20 @@ extension Issue {
   /// Use this function if, while running a test, an issue occurs that cannot be
   /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
   /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
+  @_disfavoredOverload
   @discardableResult public static func record(
     _ comment: Comment? = nil,
     sourceLocation: SourceLocation = #_sourceLocation
   ) -> Self {
     record(comment, severity: .error, sourceLocation: sourceLocation)
   }
 
-  /// Record an issue when a running test fails unexpectedly.
+  /// Record an issue when a running test and an issue occurs.
   ///
   /// - Parameters:
   ///   - comment: A comment describing the expectation.
-  ///   - severity: The severity of the issue.
+  ///   - severity: The severity level of the issue.  This factor impacts whether the
+  ///     issue constitutes a failure.  The default is .error.
```

Comment:
Better to tell developers what level constitutes a failure:
```suggestion
  ///   - severity: The severity level of the issue.  The testing library marks the test as failed if the severity is greater than `.mildInconvenience`.  The default is `.error`.
```

Reply:
I couldn't get the line wraps correct in the GitHub UI, sorry.